src/modules/components/timeline/day-item/day-item.scss

```diff
@@ -1,6 +1,3 @@
-
```

Comment:
@fabiotisci I have double checked this SCSS file and the extra style has been removed. The latest file changes do not have any extra CSS unless it is from Trevor's PDK.